src/DownloadFormat/DownloadFormat.styles.ts

```diff
@@ -0,0 +1,74 @@
+import styled, { css } from 'styled-components';
+
+interface OptionProps {
+  variant?: string;
+}
+
+export const FormatOption = styled.div<OptionProps>`
+  text-align: center;
+  font-weight: bold;
+  height: 36px;
+  font-size: 12px;
+  line-height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &:first-child {
+    border-radius: 5px 5px 0 0;
```

Comment:
Can we get this value from the theme? `p.theme.borderRasius.small | default | large`